server/src/project/geometry/geometry.service.ts

```diff
@@ -18,7 +18,7 @@ const METERS_TO_FEET_FACTOR = 3.28084;
 const QUERIES = {
   DTM_BLOCK_CENTROIDS: `
     SELECT the_geom, the_geom_webmercator, cartodb_id, concat(borocode, LPAD(block::text, 5, '0')) as block 
-    FROM dtm_block_centroids_v20201106
+    FROM dof_dtm_block_centroids_deduped_by_boroblock
```

Comment:
We'll have to come up with a solution that doesn't require us making a custom one-off table in Carto. The `dof_dtm_block_centroids` table is updated by GIS automatically and we don't want to introduce new table that we have to manually update. Moreover, I confirmed with Matt Croswell that block numbers aren't intended to be unique within a given borough.

Reply:
I don't understand, if block numbers aren't unique within a given borough, wouldn't that mean that two lots could have the same BBL?

Reply:
That was my first question but no. You can have two physical blocks with the same block number but they won't re-use lot numbers. You might have two blocks in one borough that are block "5", with three lots on each, but one would have lots 1, 2, 3 and the other would have lots 4, 5, 6, etc.

Reply:
Honestly this might be rare enough that we don't have to worry about this bug unless it's actually blocking our ability to get radius search working. If the impact is just that users see two dots on the map for projects that happen to overlap with these duplicate blocks, it's not the worst thing.

Reply:
The issue with seeing two dots is that it undermines the credibility of the radius search.  When you draw a circle that contains two dots and only one result shows up, it just looks like the filter is broken anyway.

I tried and failed to figure out how to make the SQL only take 1 of each unique concatenated "block", which would solve the issue, but wasn't able to make it work.  That would fix the problem without having to create a new table.